src/System.Runtime.Serialization.Xml/tests/project.json

```diff
@@ -32,13 +32,9 @@
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0040"
   },
   "frameworks": {
-    "netstandard1.3": {}
+    "netstandard1.7": {}
   },
   "supports": {
-    "coreFx.Test.netcoreapp1.0": {},
-    "coreFx.Test.net46": {},
-    "coreFx.Test.net461": {},
-    "coreFx.Test.net462": {},
-    "coreFx.Test.net463": {}
+    "coreFx.Test.netcoreapp1.1": {}
```

Comment:
Yes currently it is corefx.Test.netcore50 we will eventually get uap10.1 supported.